### ISSUE TITLE
Ability to subscribe to GitHub Discussions

### ISSUE THREAD

**Issue by ColinKerr** [integrations/microsoft-teams#84]
We would like to be able to subscribe to discussions similar to how you can subscribe to issues

We currently use this app to pull issues into a teams channel so we can track them along with the rest of our conversations.  We'd like to be able to track discussions in the same way.

**Comment by ashokirla** [comment]
@ColinKerr Thanks for the suggestion. Yes, support for discussions is in our radar. However, we are currently working on few other priority items like GH Actions and GHES/GHAE support. So we will look into this feature ask in Q3 CY21.

**Comment by tmarkovski** [comment]
+1 for this feature, would be great to have it!

**Comment by falnyr** [comment]
+1 This would be great with an option to choose a topic :raised_hands:

**Comment by johannesbjorkdahl-decerno** [comment]
+1 We're trying to move over conversations from teams to github discussions to create a more usable knowledge base. This would make it a lot more usable since we still need to use teams for other conversations.

**Comment by cinch-toli** [comment]
👌 Would be interested in being notified for new activity in discussions too. Is this still on the roadmap? We are currently trying out Discussions for various use cases. And having visibility of new activity was one of the first things that popped up as a suggestion to keep the discussions prompt and alive

**Comment by minhho-itr** [comment]
MSTeams are able to subscribe to discussions and when an answer is marked as answer. The ability to subscribe to the replies would be fantastic.

**Comment by rmh54** [comment]
+1 would be great to received notifications of discussion comments in the same way we receive them for issues

**Comment by sonomirco** [comment]
@ashokirla is still this feature on the radar?
Q3 Y21 is passed 🤣

**Comment by lsilverton-shell** [comment]
Another +1 for this feature! 

Getting MS Teams notifications for GitHub discussion updates/replies would be amazing and really help with our transition from other tooling.

**Comment by deepikamnaik** [comment]
Yes. +1 for this feature. It would be great to get the notifications on MS Teams.

**Comment by nazra-a** [comment]
+1 for this feature, it would be great to get GitHub discussion notification on MS Teams.
